middleware/ordered_group.go

```diff
@@ -19,13 +19,16 @@ type ider interface {
 // orderedIDs provides an ordered collection of items with relative ordering
 // by name.
 type orderedIDs struct {
-	order relativeOrder
+	order *relativeOrder
 	items map[string]ider
 }
 
+const baseOrderedItems = 5
```

Comment:
What do you think of incrementing this to 10? In general, I think at least 10+ middlewares are in play.